docs/en/commands/element/find-element.md

```diff
@@ -0,0 +1,111 @@
+# Find Element
```

Comment:
why do we have both `element.md` and `find-element.md`? likewise with the multiple variant

Reply:
That was a leftover artifact. The generation wasn't cleaning the folder. I'll add a `rimraf` to that.